apps/web/pages/index.tsx

```diff
@@ -31,19 +36,26 @@ const Home = () => {
           <p className="mt-2 text-sm text-gray-700">A list of all the posts</p>
         </div>
         <div className="mt-4 sm:mt-0 sm:ml-16 sm:flex-none">
-          <button
+          {!displayForm && <button
             type="button"
             className="inline-flex items-center justify-center rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2 sm:w-auto"
-            onClick={() => {
-              setForm(!displayForm);
-            }}
+            onClick={handleSetForm}
```

Comment:
[nit] For short handlers, feel free to use inline arrow functions
```suggestion
            onClick={() => setForm(!displayForm)}
```